draft-sharabayko-mops-srt.md

```diff
@@ -2086,15 +2086,583 @@ and RTTVar are updated by the socket based on algorithms for the sender
+from the receiving side through acknowledgement (ACK) packets
+({{ctrl-pkt-ack}}). Every 10 milliseconds the sender receives
+the latest values of RTT, RTT variance, available buffer size,
+receiving rate and estimated link capacity. Upon reception of
+an NAK packet ({{ctrl-pkt-nak}}) the sender detects packet loss
+during the transmission. These mechanisms provide a solid background
+for various congestion control algorithms integration in the
+SRT protocol.
+
+Given that SRT can operate in live and file transfer modes, there are
+two groups of congestion control algorithms defined in SRT: live
+congestion control (LiveCC) and file transfer congestion control
+(FileCC).
+
+## SRT Packet Pacing and Live Congestion Control (LiveCC) {#liveCC}
+
+To ensure smooth video playback of a receiving peer, SRT in live mode
+({{transmission-mode-live}}) must control the sender's buffer level to
+prevent overfill and depletion. Pacing control module is
+designed to send packets as fast as they are submitted by the
+application while maintaining relatively stable buffer level.
+While this looks like a simple problem, the sender's constituent
+of the Automatic Repeat Request (ARQ) system between
+input and output of the SRT sender adds some complexity.
+
+SRT needs a certain amount of bandwidth overhead in order to have
+space to insert retransmitted packets without affecting the SRT
+sender's main stream output rate too much so that packets can be
+transmitted normally.
+
+It is done by adjusting the maximum allowed bandwidth MAX_BW
+({{maxbw}}) which limits the bandwidth usage by SRT and is used
+by live congestion control (LiveCC) module to calculate the minimum
+inter-sending time of consecutive packets PKT_SND_PERIOD.
+In principle, the space between packets determines where
+retransmissions can be inserted, and the overhead represents the
+available margin. There is an empiric calculation that defines the
+interval, in microseconds, between two packets to give a certain
+bitrate. It's a function of the average packet payload (which
+includes video, audio, etc.) and configured maximum bandwidth MAX_BW.
+See {{default-liveCC}} for details.
+
+In live mode, it is allowed to drop those packets that can not be delivered
+in time ({{too-late-packet-drop}}).
+
+To sum up, a combination of pacing control and live
+congestion control (LiveCC) modules controls SRT packets pacing based
+on the input rate and an overhead for packets retransmission in order
+to avoid congestion during fluctuations of the source bitrate.
+
+In addition to these mechanisms, in highly variable networks live mode
+fairness can be accomplished outside SRT by controlling the bitrate
+of the encoder at the input of the SRT sender. SRT provides a variaty
+of network related statistics such as RTT estimation, the level of
+packet loss, the number of packets dropped by SRT, etc., which can be
+used for making decisions and adjusting the bitrate in real time.
```

Comment:
```suggestion
In live mode over highly variable networks, fairness can be achieved by controlling 
the bitrate of the source encoder at the input of the SRT sender. SRT provides a 
variety of network related statistics, such as RTT estimation, packet loss levels, 
the number of packets dropped, etc., which can be used for making decisions 
and adjusting the bitrate in real time.
```

Reply:
**In case of live streaming** over highly variable networks, fairness can be achieved by controlling
the bitrate of the source encoder at the input of the SRT sender. SRT provides a
variety of network related statistics, such as RTT estimation, packet loss **level,**
the number of packets dropped, etc., which can be used for making decisions
and adjusting the bitrate in real time.

Is there a way to say "in case of live streaming" in other words?

Reply:
Your changes are fine. Again, subjectively, there is the use of "in case...". You could also say "During live streaming . . ." or "When live streaming . . .".
Also, the word "estimation" usually refers to the act of performing a calculation where "estimate" is the result. This is a very subtle distinction, and using "estimation" as a noun is technically not wrong. More of a "common usage" thing.